CHANGELOG.md

```diff
@@ -1,5 +1,78 @@
+### Features
+
+* Kubelet Serving Certificate Rotation is now enabled by default in Sovereign cloud regions. Existing node pools in these regions will have KSCR enabled by default when they perform their first upgrade to any Kubernetes version 1.27 or greater. New node pools on Kubernetes version 1.27 or greater will have KSCR enabled by default. For more information, see [Kubelet Serving Certificate Rotation](https://aka.ms/aks/kubelet-serving-certificate-rotation). Track regional rollout at [#4986](https://github.com/Azure/AKS/issues/4986).
+* AKS now allows the use of unsupported GPU vm sizes after skipping gpu driver installation. If a GPU vm size is not in our list of supported vm sizes, we do not install the necessary gpu components or provide support. For more information, see [Skip GPU drivers](https://aka.ms/aks/skip-gpu-drivers).
+* AKS now automatically reimages all node pools in the cluster when you update the HTTP proxy configuration on your cluster using the `az aks update` command. You can use Pod Disruption Budgets (PDBs) to safeguard disruption to critical pods during reimage.
+* Advanced Container Networking now enforces Layer 7 (L7) policy restrictions in Cilium network policies. This is achieved through validating admission policies and bindings. When an L7 policy is applied without the feature enabled, the policy application will be denied and display a message to help identify the misconfiguration.
+* AKS now by default applies dynamic sizing logic for ama-logs and ama-metrics requests in production regions for AKS Automatic. This enhancement streamlines resource allocation, cost optimization, and scalability for monitoring workloads.
+* Customers in ussec and usnat regions will start using ama-logs MSI mode when they create new clusters, providing enhanced functionality. Existing clusters are not impacted. This follows the deprecation announcement for legacy authentication mode.
+
+### Behavioral Changes
+
+* Istio revision asm-1-27 is now available for the Istio-based service mesh add-on. Customers can follow canary upgrade guidance to adopt the new revision. Note that native sidecar mode is enabled by default starting asm-1-27. For full details, see the [Istio 1.27 release notes](https://istio.io/latest/news/releases/1.27.x/) and [Native sidecar mode for Istio-based service mesh add-on](https://learn.microsoft.com/azure/aks/istio-native-sidecar).
+* Envoy filters are now supported in the Istio add-on for AKS, enabling advanced traffic customization. Learn more at [istio add-on envoy filter](aka.ms/istio-add-on-envoy-filter).
+* Cluster Autoscaler will delete nodes that encounter provisioning errors/ failures immediately, instead of waiting for the full max-node-provision-time defined in the [cluster autoscaler profile](https://learn.microsoft.com/azure/aks/cluster-autoscaler?tabs=azure-cli#cluster-autoscaler-profile-settings). This change significantly reduces scale-up delays caused by failed node provisioning attempts.
```

Comment:
```suggestion
* Cluster Autoscaler will delete nodes that encounter provisioning errors/failures immediately, instead of waiting for the full max-node-provision-time defined in the [cluster autoscaler profile](https://learn.microsoft.com/azure/aks/cluster-autoscaler?tabs=azure-cli#cluster-autoscaler-profile-settings). This change significantly reduces scale-up delays caused by failed node provisioning attempts.
```